api/app/Models/ArticleMeta.php

```diff
@@ -42,4 +44,25 @@ public function article()
     {
         return $this->belongsTo(Article::class, 'article_id', 'article_id');
     }
+
+
+    /**
+     * @param Builder $query
+     * @param BaseModel $parent
+     * @return Builder
+     */
+    protected function attachParentModelToQuery(Builder $query, BaseModel $parent)
+    {
+        $query->where('article_id','=',$parent->article_id);
+        return $query;
+    }
+
+    /**
+     * @param BaseModel $parent
+     * @return void
+     */
+    public function attachParent(BaseModel $parent)
+    {
+        $this->article_id = $parent->article_id;
```

Comment:
:+1: 
Yeah, I've just totally forgot to use a relation. It will make it much easier.